homeassistant/components/smarttub/binary_sensor.py

```diff
@@ -83,12 +82,10 @@ def is_on(self) -> bool:
         return self.reminder.remaining_days == 0
 
     @property
-    def device_state_attributes(self):
+    def extra_state_attributes(self):
         """Return the state attributes."""
-        when = datetime.now() + timedelta(days=self.reminder.remaining_days)
         return {
             ATTR_REMINDER_SNOOZED: self.reminder.snoozed,
-            ATTR_REMINDER_DATE: when.date().isoformat(),
```

Comment:
The `ATTR_REMINDER_DATE` constant is now unused and should be removed as well.

Reply:
Good catch, removed